src/algebra/algebra.jl

```diff
@@ -1,5 +1,36 @@
 otimes(x::AbstractVector, y::AbstractVector) = x * y'
 otimes(x::AbstractVector) = otimes(x, x)
+
+"""
+Tensors product between second order tensors
```

Comment:
can you please add the function signature at the top of the doc string? Like this:
```julia
    otimes(A,B)
    
Tensors product between second order (....)
```